src/entities/Proposal/routes.ts

```diff
@@ -404,18 +404,39 @@ export async function createProposalTender(req: WithAuth) {
   const user = req.auth!
   const configuration = validate<NewProposalTender>(newProposalTenderValidator, req.body || {})
 
-  if (!PROMOTE_PITCH_ENABLED) {
-    throw new RequestError('Forbidden', RequestError.Forbidden)
-  }
-
   await validateLinkedProposal(configuration.linked_proposal_id, ProposalType.Pitch)
   await validateSubmissionThreshold(user, SUBMISSION_THRESHOLD_TENDER)
 
+  const tenderProposals = await ProposalModel.getProposalList({
+    linkedProposalId: configuration.linked_proposal_id,
+    order: 'ASC',
+  })
+
+  if (
+    tenderProposals.find(
+      (proposal) => proposal.status === ProposalStatus.Passed || proposal.status === ProposalStatus.Rejected
+    )
+  ) {
+    throw new RequestError('Pitch proposal already went through the tender process')
+  }
+
+  const hasStartedTenderProcess = Time(tenderProposals[0].start_at).isBefore(Time())
+  if (hasStartedTenderProcess) {
+    throw new RequestError('Tender process already started for this pitch proposal')
+  }
+
+  const start_at =
```

Comment:
This for me is a hint that we might be needing a new abstraction for this... 

For example, a `TenderingProcess` object, with a `start_at`, and `end_at`, that we can associate to a pitch proposal and/or it's tenders, and use it to make this decisions (instead of having to use each tender, and having to get the associated tenders list)